alf/examples/ppo_rnd_mrevenge.gin

```diff
@@ -7,6 +7,12 @@ include "image_game_model.gin"
 import alf.algorithms.rnd_algorithm
 import alf.tensor_specs
 
+# FrameStacker data transformer does not work with PPO because FrameStacker needs
+# ReplayBuffer, while PPO uses "one_time" experience replay, which does not use
+# ReplayBuffer. So we need to use FrameStack gym wrapper instead.
+suite_gym.load.gym_env_wrappers=(@DMAtariPreprocessing, @FrameStack)
+FrameStack.stack_size=4
```

Comment:
It's easy to make mistakes given the difference between PPO and other algorithms. For one_time replay buffer, is it possible to always maintain extra stack_size-1 frames when clearing the buffer?

Reply:
The current replay buffer does not support that.

Reply:
I mean the OneTimeExperienceReplay class can easily do that. But it seems difficult to integrate it with the current code in FrameStacker.